docs/UserGuide.md

```diff
@@ -727,7 +738,7 @@ Examples:
 
 #### Editing a lesson: `ledit`
 
-Edits the specified lesson of the specified student in TAB with the indicated changes for specified fields.
+Need to update lesson information? This command allows you to do just that.
```

Comment:
Do the same for the rest of the commands as well. Otherwise, not very consistent in tone. If not, I think its fine to leave it as the original one